canonical-web-and-design/canonicalwebteam.launchpad
# v0.2.0: Combine classes into one Launchpad class

As explored a little here:
https://github.com/canonical-web-and-design/snapcraft.io/pull/2555#issuecomment-589999414

QA
--

See tests pass. I'll also fix up https://github.com/canonical-web-and-design/snapcraft.io/pull/2555 to use this branch if you wanna check that.
